### DISCUSSION TITLE
bam index files

### DISCUSSION THREAD

**Discussion by lerch-a** [malariagen/vector-data#19] [Q&A]
Great job on the manual! 
I would love to have access to the bai files (bam index files). Are they available somewhere? 
I'd like to download aligned sequence read only for a specific marker region. This would allow phasing or microhaplotype calling for a small region without the need of downloading the whole bam file.
Thanks

**Comment by alimanfoo** [comment]
Hi @lerch-a, thanks for the kind words!

This isn't in the user guide but bam and bai files are also available from google cloud. E.g., files for sample AR0001-C in sample set AG1000G-AO:

* https://storage.googleapis.com/vo_agam_release/v3/alignments/AG1000G-AO/AR0001-C.bam
* https://storage.googleapis.com/vo_agam_release/v3/alignments/AG1000G-AO/AR0001-C.bam.bai

I don't know whether we can afford to keep all the BAMs in google cloud indefinitely, which is why they aren't in the documentation, but they'll be there at least for the next few months and you're welcome to access them while they are.

Hth.

**Comment by lerch-a** [comment]
It is sufficient to only keep the bai files on the google cloud. As you can make a partial download for example with:

samtools view -bho out.bam -X ftp://ftp.sra.ebi.ac.uk/vol1/ERZ169/ERZ1695275/AR0001-C.bam ~/Downloads/AR0001-C.bam.bai 3L:1000-1010

Maybe this example could go into the guide. Not many people are aware that you can do this.

Thanks

**Reply by alimanfoo** [reply]
Thanks @lerch-a, that's a great suggestion.
